lib/reporters/xunit.js

```diff
@@ -158,6 +158,7 @@ XUnit.prototype.test = function (test) {
   var attrs = {
     classname: test.parent.fullTitle(),
     name: test.title,
+    file: test.file,
```

Comment:
Working with Cypress, I had to change this to:

```suggestion
    file: test.file || test?.invocationDetails?.absoluteFile,
```

Reply:
Can you explain where exactly `invocationDetails?.absoluteFile` comes from? Is this a Cypress-specific object/property?

I'd be concerned with adding Cypress-specific logic inside the reporter, if that is the case. Why should Cypress, or any other third-party library, be treated specially?

If it's a standard property, let me know. If it's not, could Cypress switch to the standard property, or is there some other workaround?

Reply:
cc @villelahdenvuo, ping?